features/step_definitions/read_entire_post.rb

```diff
@@ -0,0 +1,47 @@
+Given /^I am on the home page$/ do
+  @page = CreationPage.new @browser
+  @page.goto
+  @page.submit_two_posts
+  @home = ListingPage.new @browser
+  @home.goto
+  @reading = ReadingPage.new @browser
+end
+
+When /^I click on post title$/ do
+  @reading.goto
```

Comment:
You aren't clicking the post title like the step suggests. Instead, you're navigating straight to the pre-defined URL.